compose/ingress/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM nginx:latest
+
+COPY entrypoint.sh /entrypoint.sh
+COPY nginx.conf /etc/nginx/conf.d/default.conf
+
+# COPY certs/selfsigned.crt /etc/nginx/ssl/selfsigned.crt
+# COPY certs/selfsigned.key /etc/nginx/ssl/selfsigned.key
```

Comment:
Since `/etc/nginx/ssl/selfsigned.key` is created by the entrypoint script, good to remove these comments?